pages/common/age.md

```diff
@@ -0,0 +1,28 @@
+# age
+
+> A simple, modern and secure file encryption tool.
+> More information: <https://age-encryption.org>.
+
+- Generate an encrypted file that can be decrypted with a passphrase:
+
+`age --passphrase --output {{path/to/encrypted_file}} {{path/to/unencrypted_file}}`
+
+- Generate a keypair, saving the private key and the public key to an (unencrypted) keyfile, and printing the public key to stdout:
+
+`age-keygen --output {{path/to/keyfile}}`
+
+- Encrypt a file to one or several public keys that are entered as literals:
+
+`age --output {{path/to/encrypted_file}} --recipient {{publickey1}} --recipient {{publickey2}} {{path/to/unencrypted_file}}`
```

Comment:
```suggestion
- Encrypt a file with one or more public keys:

`age --output {{path/to/encrypted_file}} --recipient {{public_key1}} --recipient {{public_key2}} {{path/to/unencrypted_file}}`
```

Reply:
I'm not too sure about those… Is encrypting "with" a public key indeed better than "to"? Perhaps it might be easier to understand by new users, perhaps not… And I'd rather like to keep the "as literals" part in order to make clear that these aren't supposed to be files… YMMV, of course.

Reply:
I implemented some of your suggestions now, just not the ones I expressed concerns with above. Let me know what you think.

Reply:
Hmm, that sounds ok to me.  "Encrypt a file to a key" sounds odd to me, but that might be because I'm not that familiar with encryption myself, having done little of it.  If you're confident that it's technically correct, I'm perfectly fine with about it. 

About "as literals" - I know what you mean; it is a possible concern, but we normally add ornaments such as `path/to/`, file extensions, or something like `public_key_file` to denote that it is a file and not a string.  I don't mind the current wording myself, but I searched our 2,339 English pages for "literal" and only found 6 pages...but that doesn't mean we can't do it now.

@sbrl, could we get some advice on this?

Reply:
Hey there! I agree with @bl-ue here that `Encrypting to a key` really doesn't make a lot of sense here. `with` indicates that it uses  the public key in the process, whereas `to` would imply that it converts the input _into_ a public key, which does not make sense.

Where does it get the public keys from? If it's the id fo a key it pulls say from GPG, then `public_key_id_1` and `public_key_id_2` would be more appropriate. If the public key is stored in a file on disk that is referenced, then prefixing it with `path/to/` would be preferable to ensure consistency with other tldr pages.

Reply:
It's the literal Base64 encoded public key. Ed25519 ones are short enough to do this. Age is designed as a partial replacement for PGP/GPG, and has nothing to do with those. For specifying a file with the public keys, look at the example below.

Reply:
> Public key encryption, in which a message is encrypted with a recipient's public key.

https://en.wikipedia.org/wiki/Public-key_cryptography#Description

Wikipedia seems to agree with you two, btw. I'll just change it to that, then.

Reply:
@sbrl What I could do is specify that it should be Base64 encoded (as was done in Minisign), however, that's the format one would always get Age keys in, thus making that perhaps a bit redundant, and it's also not really mentioned in the official instructions (both in the GitHub README and in `age --help`)… Perhaps @FiloSottile would like to chime in here?

Reply:
Cool, thanks.

Hrm. Given that this command both generates and uses keypairs, I would suggest that the detail that the keys are stored as base64 isn't something that needs to be mentioned in the tldr page. Remember that the page is supposed to serve as a quick reference guide - it's not meant to be exhaustive :-)

Reply:
Users are not expected to think about the encoding of the recipients (and it's technically not Base64), so I would avoid mentioning it.